pemi/data_subject.py

```diff
@@ -47,11 +47,12 @@ def validate_schema(self): #pylint: disable=no-self-use
 
 
 class PdDataSubject(DataSubject):
-    def __init__(self, df=None, **kwargs):
+    def __init__(self, df=None, strict_match=False, **kwargs):
```

Comment:
I feel like "strict_match" isn't obvious enough.  What about one of these:

strict_match_schema=False
schema_columns_only=False

Reply:
Agree. strict_match_schema would be more obvious